src/evict/evict_lru.c

```diff
@@ -2058,9 +2058,13 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
          * not yet globally visible, eviction will fail. This heuristic avoids repeated attempts to
```

Comment:
We should also change the comment

Reply:
Fixed in fec7ccb73be21984ca38a5d3926b409397c7a699